src/app/components/videochat/jh-video-chat.html

```diff
@@ -19,7 +19,7 @@
   </header>
 </div>
 
-<div class="row">
+<div class="row" id="body">
```

Comment:
Not sure if I like "body" as an id for something that is actually just part of a directive. Sounds like the body of the full page.